### PR TITLE
Add scrolling mozilla marquee to hero unit.

### PR DESCRIPTION
**Pull request by toolness**
This is an attempt at implementing @hannahkane's suggestion in https://github.com/mozilla/teach.webmaker.org/issues/401#issuecomment-86035607.

mozilla-marquee

The logo appears smaller in Firefox, but the effect is similar.

I'm also pleased to see that JSX/React allows the use of the `<marquee>` tag, and browsers still support it, albeit inconsistently.

### CODE REVIEW

File: components/hero-unit.jsx [hunk 1]
```diff
@@ -34,6 +34,12 @@ var HeroUnit = React.createClass({
         <div className="col-md-12 hero-unit" style={{
           backgroundImage: 'url(' + this.state.image + ')'
         }}>
+          <marquee>
+            <span style={{color: 'transparent'}}>
+              MOZILLAAAAAAAAA!!
+            </span>
```
Comment (toolness) [inline components/hero-unit.jsx]: For some reason this `<span>` is needed in Firefox for the `` below it to show. Regardless, it provides useful semantic content for screen readers and search engines.

**Comment by iamjessklein** [comment]
R+ :shipit: :anchor:

**Comment by davidascher** [comment]
Is there a reason you didn't belt-and-suspenders it with a `<blink>` tag?

**Comment by toolness** [comment]
Ahh my bad @davidascher, I totally should've done that :)

**Comment by LauraHilliger** [comment]
I feel like it should star wipe out, actually.

**Comment by thisandagain** [comment]
:star2: :star2: :star2: :star2: :eyes:

**Comment by hannahkane** [comment]
It's really refreshing to be on a team that takes my ideas seriously. Can we please play a Maroon 5 song automatically on page load?

**Comment by iamjessklein** [comment]
for you @hannahkane anything. Does "Sugar" work for you?

**Comment by toolness** [comment]
@hannahkane, I'm happy to play a song automatically on page load, but I have concerns about Maroon 5 because their music is copyrighted and we might get a [DMCA Takedown](http://www.dmca.com/FAQ/What-is-a-DMCA-Takedown) notice. Do you happen to like anything Creative Commons licensed, like, say, Jonathan Coulton's rendition of [Baby Got Back](https://www.youtube.com/watch?v=MCWaN_Tc5wo) or [Code Monkey](https://www.youtube.com/watch?v=qYodWEKCuGg)?

**Comment by jgmac1106** [comment]
Auto play video and songs? What's next a big flash video pre-roll?  Think
about GLAMs who may visit the site over and over again in very public
places.

**Comment by toolness** [comment]
I would totally approve of a flash video pre-roll if it weren't proprietary! But video, that's a good idea... We could maybe have the [Choose Independent](https://www.youtube.com/watch?v=LtOGa5M8AuU) video playing in the hero unit, or the video for one of the Coulton songs...

**Comment by ScottDowne** [comment]
Looks good to me.

**Pull request merged**